Pipfile

```diff
@@ -0,0 +1,31 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+arrow = ">=0.7.0"
+beautifulsoup4 = ">=4.3.2"
+flask = ">=0.10.1"
```

Comment:
Unrelated to this Pull Request… but wow we should get Møte up to Flask 1.1.x.

Reply:
@jwflory If you could create a ticket for it, I'll be more than happy to port it :)

Reply:
There you go man https://github.com/fedora-infra/mote/issues/138, :smile:

Reply:
But fulfilling the requirements would get you 1.1.x by default as the requirements are leniently laid down. It simply asks for a version above 0.10.1. We should more likely update the code to incorporate versions greater than. say, 1.0 maybe.

Reply:
+1. Thanks for opening an issue to track this! :+1: